src/components/customComponents/HeaderNavBar/index.js

```diff
@@ -1,9 +1,12 @@
 import {Link} from 'react-router-dom';
 import { useCart } from '../../../contexts/cart-context';
+import { useWishlist } from '../../../contexts/wishlist-context';
 
 export const HeaderNavBar = () => {
     const { state } = useCart();
+    const {wishlistState} = useWishlist();
```

Comment:
Since you are following the approach of using the state and getting the lengths out of it, it will be better to follow same naming convention. Name the cart's `state` as `cartState`

Reply:
Thanks for pointing it out. Corrected the naming.